aries-site/src/layouts/main/Header.js

```diff
@@ -23,11 +22,27 @@ const StyledHeader = ({ ...rest }) => {
       {...rest}
     >
       <Link href="/" passHref>
-        <AppIdentity
-          brand="hpe"
-          logoOnly={size === 'small' && searchFocused}
-          title="Design System"
-        />
+        <Button plain>
+          <Box
+            direction="row"
+            align="center"
+            gap="medium"
+            // pad maintains accessible hit target
+            // non-responsive maintains same dimensions for mobile
+            pad={{ vertical: 'small' }}
+            responsive={false}
+          >
+            <Hpe color="brand" />
+            {(size !== 'small' || (size === 'small' && !searchFocused)) && (
+              <Box direction="row" gap="xsmall">
+                <Text color="text-strong" weight="bold">
+                  HPE
+                </Text>
+                <Text color="text-strong">App Name</Text>
```

Comment:
While we're here, can we play with `<Text wordBreak="keep-all" ... />` or a no wrap on the Box? I've never been a fan of when the name does this:
![Screen Shot 2020-11-02 at 11 07 38 AM](https://user-images.githubusercontent.com/1756948/97903311-108dd900-1cfc-11eb-8efb-3856551cad43.png)

Reply:
Agreed. Let me look into that.

Reply:
The best implementation I could do was wrapping the text to appear as such:
<img width="882" alt="Screen Shot 2020-11-04 at 4 41 54 PM" src="https://user-images.githubusercontent.com/12522275/98183838-e257f700-1ebd-11eb-942d-32db6e06bdb6.png">